semrush/intergalactic
# Update skeleton chart

Updated all chart skeletons and renamed one component.

## Motivation and Context

Old charts didn't stretch when resizing.  Changed from svg to div with background

## How has this been tested?

- See screenshots test
- See example in website
- Check work in Chrome, Safari and FF

## Screenshots (if appropriate):
<img width="517" alt="Screenshot 2023-07-12 at 15 20 20" src="https://github.com/semrush/intergalactic/assets/4582799/9639f382-98e6-4e56-b7d4-ae595228cf4c">

## Types of changes



- [x] Nice improve.

## Checklist:




- [x] I have updated the documentation accordingly or it's not required.
- [x] Unit tests are not broken.
- [x] I have added changelog note to corresponding `CHANGELOG.md` file with planned publish date.
- [x] I have added new unit tests on added of fixed functionality.
